extensions/vscode/webviews/homeView/src/components/configErrors/InvalidTOMLFile.vue

```diff
@@ -0,0 +1,44 @@
+<!-- Copyright (C) 2024 by Posit Software, PBC. -->
+
+<template>
+  <div v-if="error" class="error-block">
+    Configuration Error!
+    <ul class="bullets">
+      <li>Invalid TOML</li>
+      <li>
+        {{ error.data.problem }}
+      </li>
```

Comment:
This is coming directly from the backend, yeah? Or do we process it somewhere?

Asking because I'm not sure if errors like the following are all that helpful for folks most likely to see them:

```
cannot decode TOML
string into struct field config.Config.Files of type []string
```

Reply:
This is being returned from the TOML parser go package we use as the error encountered. We "could" consider translating / mapping it to another error, but it would be a maintenance issue when the package changes.